app/views/management/schools/show.html.erb

```diff
@@ -49,6 +49,8 @@
 
 <%= render 'management/schools/record_intervention', school: @school %>
 
+<%= render 'schools/dashboard/transport_surveys', school: @school %>
```

Comment:
I think for the moment we were only proposing to add this to the pupil dashboard, and not the management dashboard. So can you remove this?